### ISSUE TITLE
Unable to scroll DropdownButton items using VoiceOver

### ISSUE THREAD

**Issue by kennethj** [flutter/flutter#91032]
The `DropdownButton` menu does not scroll when using a VoiceOver. When VO reaches the bottom of the visible list it announces "Dismiss" instead of scrolling to the next item.

## Steps to Reproduce

- Open `DropdownButton` menu
- Use VO to navigate through items until you reach the bottom

``` dart 
import 'package:flutter/material.dart';

main() async {
  runApp(const MyApp());
}

class MyApp extends StatefulWidget {
  const MyApp({Key key}) : super(key: key);

  @override
  _MyAppState createState() => _MyAppState();
}

class _MyAppState extends State<MyApp> {
  String message = "";
  @override
  Widget build(BuildContext context) => MaterialApp(
        home: Center(
          child: Material(
            child: DropdownButton(
              onChanged: (v) {},
              isExpanded: true,
              items: [
                for (var i = 0; i < 100; i += 1)
                  DropdownMenuItem(
                    child: Text("test #$i"),
                    value: i,
                  )
              ],
              value: 1,
            ),
          ),
        ),
      );
}
```

**Expected results:** 
VoiceOver should scroll the list and navigate to the next item until it reaches the end of the items.

**Actual results:** 
VoiceOver instead navigates out to the menu once it reached the end of the visible items.

![VO-menu](https://user-images.githubusercontent.com/862054/135482053-69c679e3-0ee2-4b96-967e-727ffb7df373.gif)

<details>
  <summary>Logs</summary>
```
[✓] Flutter (Channel stable, 2.5.0, on macOS 11.6 20G165 darwin-x64, locale en-US)
    • Flutter version 2.5.0 at /Users/dev/development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (3 weeks ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/dev/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /Library/Java/JavaVirtualMachines/jdk-11.0.5.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment 18.9 (build 11.0.5+10-LTS)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5.1, Build version 12E507
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/dev/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7678000/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)


[✓] Android Studio (version 2020.3)
    • Android Studio at /Users/dev/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7678000/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] IntelliJ IDEA Ultimate Edition (version 2021.2.2)
    • IntelliJ at /Users/dev/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] IntelliJ IDEA Ultimate Edition (version 2021.1.1)
    • IntelliJ at /Users/dev/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/211.7142.45/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.3)
    • IntelliJ at /Users/dev/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/203.7717.56/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.60.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.26.0

[✓] Connected device (4 available)
    • Pixel 5 (mobile)               • 0A221FDD4003PL                           • android-arm64  • Android 11 (API 30)
    • iPhone (mobile) • 08f9472d07522760b228b786dd9856b8f853b427 • ios            • iOS 14.7.1 18G82
    • macOS (desktop)                • macos                                    • darwin-x64     • macOS 11.6 20G165 darwin-x64
    • Chrome (web)                   • chrome                                   • web-javascript • Google Chrome 94.0.4606.61

```

</details>

**Comment by kennethj** [comment]
I may have already file this bug https://github.com/flutter/flutter/issues/84719
This may be a duplicate.

**Comment by kennethj** [comment]
This issue is a duplicate of #84719 and appears to be fixed in version 2.6 in the current beta channel
